dev/reference/tools/tidb-lightning/glossary.md

```diff
@@ -0,0 +1,187 @@
+---
+title: TiDB Lightning Glossary
+summary: List of special terms used in TiDB Lightning.
```

Comment:
How about adding these TiDB Lightning terms to the existing `glossary.md` at the root directory? (`docs/dev/glossary.md`) I think maintaining one glossary would be friendlier to users. They can easily search all terms in one place.

Reply:
🤔 pingcap/docs-cn#2066 (DM) placed it separately already, and https://github.com/pingcap/docs-cn/pull/2102#pullrequestreview-327970130 (Binlog) is pending change. WDYT @pingcap/ecosystem-tools.

Reply:
How about adding Lightning & Binlog terms to the existing `glossary.md`, but keeping DM separately.

Reply:
@csuzhangxc Yes. DM will have its own separate TOC later on the website, so we can maintain a separate glossary for DM.

Reply:
I don't recommend putting glossary  together for two reasons
* proper nouns are not uniformly planned
* proper nouns are more suitable in their own fields，but can be refered, such as backup and recovery toolsets, rather than uniform placement. Obviously many proper nouns only make sense in its domain

Reply:
- Words in the glossary will be written and added based on our current documentation.
- I agree with you that proper nouns only make sense in a specific field. That's why it's a proper noun and why we need to explain it. If users cannot get what it means in what context, then we need to improve that description to make it clearer.

"be unified" is never only for "unity". Instead, we want to provide one place for users to look up any word that they are not sure about its meaning.

Reply:
Let me summarize
* you concern that how to look up any word
* I concern that how user get the context (the specific field information) 

So how do we resolve it, I don't think these two goals are contradictory

Reply:
@lilin90  @GregoryIan How are we going to resolve this? This PR and pingcap/docs-cn#2102 are still waiting for the decision...

Reply:
@kennytm Changes in this PR are acceptable. We can add them to the complete Glossary by providing links later.